README.md

```diff
@@ -1 +1,48 @@
-# To-Do-List
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Personal-Portfolio
+
+This project is mainly about creating personal portfolio by HTML5 and CSS3.
+
+## Built With
+
+- Major languages: HTML5,CSS3
+- Technologies used: linters, Lighthouse, Webhint, Stylelint
+
+## Getting Started
+
+To access a local copy up and runn follow these simple steps.
+
+### Prerequisites
+
+- web browser
+
+### Usage
+
+- clone this repository into your local machine
+- unzip it
+- open the 'index.html' file with your browser
+
```

Comment:
- [ ] Kindly re-evaluate your readme documentation. I don't think this project is the Personal portfolio

- [ ] I also think we use a different approach for running the app locally, not just opening `index.html`. We need to install packages and run `npm start` command